src/tests/multihost/alltests/test_default_debug_level.py

```diff
@@ -156,28 +156,55 @@ def test_0004_bz1893159(self, multihost, backupsssdconf):
         """
         :title: default debug logs: Check default level 2
         :id: d44d5883-fc52-418d-b407-3ac63f7104d8
+        :setup:
+          1. Remove debug_level option from sssd.conf
+          2. Set domains = typo_domain (non existing) in [sssd] section
         :steps:
-          1. Remove debug_level from sssd.conf
-          2. Start sssd after clearing cache and logs
-          3. Kill pid of sssd with signal SIGUSR2
-          4. Check logs
+          1. Start sssd after clearing cache and logs
+          2. Check sssd.log contains log related to 'SSSD cannot load config'
         :expectedresults:
-          1. sssd should use default debug level with no level defined
-          2. Succeeds
-          3. sssd process is killed
-          4. logs of level of 0x0040 are in the log file
+          1. SSSD will not start
+          2. /var/log/sssd/sssd.log contains 'SSSD couldn't load configuration' log
+        """
+        section = f"domain/{ds_instance_name}"
+        domain_params = {'debug_level': ''}
+        tools = sssdTools(multihost.client[0])
+        tools.sssd_conf(section, domain_params, action='delete')
+        tools.sssd_conf('sssd', {'domains': 'some'}, action='update')
+        multihost.client[0].service_sssd('stop')
+        tools.remove_sss_cache('/var/log/sssd')
+        multihost.client[0].run_command('systemctl start sssd', raiseonerr=False)
+        log = '/var/log/sssd/sssd.log'
+        log_str = multihost.client[0].get_file_contents(log).decode('utf-8')
+        pattern = re.compile(r'SSSD couldn\'t load the configuration database')
+        assert pattern.search(log_str)
+
+    @pytest.mark.tier1_4
+    def test_bz1893159(self, multihost, backupsssdconf):
+        """
+        :title: default debug logs: default log level logs in sssd.log
+        :id: 8f9c8c47-a1f6-4ec0-b979-202d8d6dc6c3
+        :setup:
+          1. Remove debug_level option from sssd.conf
+          2. Set ldap_uri to a non-existing ldap-server
+          3. Start sssd after clearing cache and logs
+        :steps:
+          1. send SIGUSR2 signal to sssd process
```

Comment:
The kill command was deleted from the code.

Reply:
Yes, removed. SIGUSR2 would ask sssd to go online immediately. There was condition in original test to do that. The goal of testcase is to check what is being logged, with default debug_level,when sssd is forced to go offline.